File: 2112-minimum-difference-between-highest-and-lowest-of-k-scores/minimum-difference-between-highest-and-lowest-of-k-scores.py
class Solution:
    def minimumDifference(self, nums: List[int], k: int) -> int:
        nums = sorted(nums)
        i = 0
        j = k - 1
        output = 10 ** 5

        while (j < len(nums)):
            difference = nums[j] - nums[i]
            output = min(difference, output)

            i += 1
            j += 1

        return output
        

File: 2112-minimum-difference-between-highest-and-lowest-of-k-scores/minimum-difference-between-highest-and-lowest-of-k-scores.js
var minimumDifference = function(nums, k) {
    nums = nums.sort((a, b) => a - b)
    let i = 0
    let j = k - 1
    let output = Infinity

    while (j < nums.length) {
        diff = nums[j] - nums[i]
        output = Math.min(output, diff)
        i +=1
        j += 1
    }
    return output
};